views/page-home.pug

```diff
@@ -37,14 +38,19 @@ block content
 						li mufasa: 1899
 						li adam: 1882
 						li Cucumber: 1879
-				p Version 1.3.0 released 3-5-2019
-				h4 Fixed issue: Chats in replays work again.
-				h4 Returning feature: typing indicator (again). This has been overhauled and should work better.
-				h4 New player setting: disable typing indicator. If you feel this is adveresly affecting your browser, turn this off in your settings.
-				h4 New feature: Creategame overhaul.
-				p Thanks to Vigasaurus the create game page has been redone with many new looks and feels including some fun templates.
-				h4 New feature: enhanced information for grey players - they will be alerted when they try to sit in rainbow games, and alerted when they click private games only mode. Thanks to contributor samf.
-				h3 Many smaller fixes and UI updates!
+					p Version 1.4.0 released 3-27-2019
+					h4 Many Moderation Improvements -Vigasaurus
+					h4 Silent Game Features now work as intended -Vigasaurus
+					h4 Returning feature: Grey and Private-Only players are alerted when they try to join games that they can't -samf
+					h4 Many smaller fixes and minor UI changes!
+					p Version 1.3.0 released 3-5-2019

```

Comment:
I recommend cleaning this page up further by deleting lines 46-end of page (99) and emphasizing the Full Changelog section at the bottom centered as a direct link to the rest of the changelog ... perhaps??